packages/react/src/Logo/Logo.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes } from 'react'
+import { LogoAmsterdam, LogoGgdAmsterdam, LogoStadsarchief, LogoStadsbankVanLening, LogoVgaVerzekeringen } from '.'
```

Comment:
This causes a circulair dependency, you're also importing from `index.ts` which exports this file.

Reply:
Moved to subfolder brand